uspace/lib/c/arch/amd64/include/libarch/tls.h

```diff
@@ -54,7 +54,7 @@ static inline tcb_t *__tcb_get(void)
 	void *retval;
```

Comment:
Why not make this tcb_t *retval so that you don't have to cast it below?

Reply:
I didn't want to modify something that was passed to assembly (though have no problem making this change, it was only to silence warnings).

Reply:
The assembly does not know pointer types, it only cares about the address.

Reply:
Ok, thanks.